src/scss/monkey/_letters.scss

```diff
@@ -1,10 +1,123 @@
+.monkey {
+  overflow: hidden;
+}
+
+.letter-spans {
+  height: 40px;
+  margin: auto;
+  overflow-x: scroll;
+}
+
 #letters {
   font-size: 1.6em;
+  line-height: 1.2;
+  margin: auto;
+  position: relative;
+  display: table;
+  .letter {
+    cursor: pointer;
+    margin: 0px 3px;
+    border-bottom: none;
+    display: inline;
+    float: left;
+    vertical-align: top;
+  }
   .letter-active {
     color: #00b69d;
     border-bottom: 2px solid;
   }
-  span {
-    cursor: pointer;
+  #letters-container {
+    height: 25px;
+  }
+}
+
+.monkey-icons {
+  #letters {
```

Comment:
Right, fair enough. `#letters` probably shouldn't be an ID anyway, if we support multiple monkeys. My bad, I guess!